scripts/wasmtestreport.py

```diff
@@ -260,7 +258,7 @@ def test_single_file_non_deterministic(source_file, result, timeout_sec=DEFAULT_
         else:
             if retcode == 0:
                 add_test_result(result, str(source_file), "Success", None, output)
-            elif retcode == 134 or retcode == 139:
+            elif retcode in (134, 139):
```

Comment:
This check is done more than once with the same check, would it be possible to abstract it to a function